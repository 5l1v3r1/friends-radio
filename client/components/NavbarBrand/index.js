import React from 'react'
import Link from 'react-router/Link'

const NavbarBrand = ({className, ...props}) =>
  <Link className={`${className} navbar-brand`} {...props} />

export default NavbarBrand
