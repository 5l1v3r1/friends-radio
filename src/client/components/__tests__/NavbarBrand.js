import React from "react"
import {shallow} from "enzyme"

import NavbarBrand from "../NavbarBrand"

test("NavbarBrand", () => {
  const wrapper = shallow(
    <NavbarBrand className="foo" otherProps="bar" to="lol">
      Brand.
    </NavbarBrand>,
  )
  expect(wrapper).toMatchSnapshot()
})
