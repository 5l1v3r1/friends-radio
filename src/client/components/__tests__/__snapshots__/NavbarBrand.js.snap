// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarBrand 1`] = `
<Link
  className="foo navbar-brand"
  otherProps="bar"
  replace={false}
  to="lol"
>
  Brand.
</Link>
`;
